### ISSUE TITLE
Cannot Create Project (Master)

### ISSUE THREAD

**Issue by hookercookerman** [revans/Ti#27]
No such file or directory Resources/KS_nav_ui.png

**Comment by michelson** [comment]
same error here , i have clone & installed the master too, tried with ruby 1.9.3 & 1.9.2.
any hints on this ?

thanks

**Comment by mikesten** [comment]
Yep, same here. The project.rb generator looks for Resources/KS_nav_ui.png and Resources/KS_nav_views.png, which aren't present in the default project resources I'm getting out of Titanium Studio when running:
`ti new ti-gen-app uk.co.donotremove.tigenapp desktop`

I don't see anything about desktop not being supported but maybe that's my cause?

**Comment by michelson** [comment]
check if "titanium" cmd runs ok in your shell, that was my problem

**Comment by mochiz** [comment]
Did you mean, Do you using Zsh?

In my case. I added alias to .zsh_profile.

But, titnaium cmd is called from Bash.
Like this.
https://github.com/revans/Ti/blob/master/lib/ti/generate/project.rb#L83

I didn't add alias to .bash_profile.
So titanium cmd not found.

I resolved by adding an alias to the bash_profile.

**Comment by Sophrinix** [comment]
This. Thank you mochiz. It really should be spelled out in the docs put it in bash_profile (not bashrc or any other file). I also use zsh and this threw me for a loop.
